packages/hash/README.md

```diff
@@ -147,6 +123,19 @@ yarn workspace @hashintel/block-name serve
 
 See https://blockprotocol.org/docs/developing-blocks
 
+## Development
+
+[//]: # (TODO: Pointers to where to update/modify code)
+
+### The Graph Query Layer
+HASH's primary datastore is an entity graph. The service that provides this is located within the [/packages/graph](/packages/graph) folder. The README contains more information for development.
+
+> **CAUTION:** At the moment, the graph starts the services it depends on differently to the rest of the codebase. The README describes a way to start the required docker containers for development _of the graph_. 
+> 
+> **Before running HASH, it's required to take down the other docker compose, and to follow the instructions on how to run `external-services` as outlined in the [Getting Started](#getting-started) section.** 
```

Comment:
It is not. 

If you run `cargo make deployment-up` as per the graph README, you will have a collection of docker services called "Deployment" (or something similar).

If you run `yarn external-services up` you will have a collection of docker services called "external-services" (or something similar).

These two conflict with one another, as they both try and set up Postgres, Kratos, etc.

_The reason_ we have 2, is that `external-services` dockerises and starts the graph query layer, and you don't want that when you're developing the graph query layer.

This is what's alluded to with the "this is going to be improved soon while allowing non-container-based development" thing.

Reply:
I wanted to avoid explicitly saying how to take it down here because us duplicating such instructions leads to fragmentation, as they are inevitably missed (see a large portion of this README for an example).

We really want to restructure all of these (post repo restructure) to have clearly labelled and encapsulated sections that can be referred to within one another. As an example, we should define the `external-services` commands once, and refer to that everywhere we say "this requires them to be up" or "you want to take them down"

Reply:
If I read this and I don't know how to take them down, I'm going to ask someone or spend time googling it. That's wasted time, for the sake of not just sticking in 

> it's required to take down the other docker compose (`yarn external-services down`)

that seems worse than omitting the instruction because ideally we'd arrange the README somewhere else. let's just get the info in that people need in whatever scrappy format and clean it up later

Reply:
Well it's not that command... it's `cargo-make deployment-down` :(

I agree this is confusing, but this is mostly because of the current hodgepodge of READMEs all crammed into one. We don't have a clear separation of concerns.

I will point out that this is a disclaimer specifically targeting the graph development 
<img width="1035" alt="image" src="https://user-images.githubusercontent.com/25749103/194571192-1384794d-67bc-4100-8c7d-55dec03da169.png">

I think the only people who will read this, and/or need to use it, are people who have already read the Graph README, and would check there. (Where there is another callout to this)

Reply:
How about
```suggestion
> **CAUTION:** At the moment, the graph starts the services it depends on differently to the rest of the codebase. The README describes a way to start the required docker containers for development _of the graph_. 
> 
> **These containers conflict with those started by `external-services`. Before running HASH, it's required to tear down the ones started for the graph by following the instructions in the respective README.* 
```

Reply:
Then let's just delete this entire quote and just say 

> HASH's primary datastore is an entity graph. The service that provides this is located within the [/packages/graph](/packages/graph) folder. The README contains more information for development. You do not need to visit that README or folder unless you want to amend the graph service.

and leave the rest in the graph README.

Reply:
I have done so, while adding a small bit to the Graph README saying you must teardown the graph services when running `external-services`